examples/complete/variables.tf

```diff
@@ -1,4 +1,7 @@
-variable "example" {
-  type        = string
-  description = "The value which will be passed to the example module"
+variable "region" {
+  type = string
+}
+
+variable "sg_rules" {
```

Comment:
Avoid abbreviations. However, since this module is all about security group rules, I think it's safe to shorten this to `rules`